packages/components/src/InputDate/InputDate.rebuilt.tsx

```diff
@@ -83,3 +94,5 @@ export const InputDateRebuilt = forwardRef(function InputDateInternal(
     );
   }
 });
+
+InputDateRebuilt.displayName = "InputDateRebuilt";
```

Comment:
the nonstandard forward ref approach we had was causing stack traces to become hard to follow. this should address that.